cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/PipelineStopButton/PipelineStopPopover.scss

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+@import "../../../../../styles/variables.scss";
+
+.stop-btn-popover {
+  .popper {
+    z-index: 999;
+    width: 450px;
```

Comment:
Might be better left if this is the max width may be? This is what I am seeing in a screensize of 1000px width (the min we care about is probably 800px). 

If the users use this in a smaller screen size (VM) might have issues. We might want to give this a thought. Agreed pipeline detailed view is not entirely responsive. We can start somewhere. @leacd any suggestions? 

![screen shot 2018-02-28 at 6 05 49 pm](https://user-images.githubusercontent.com/1452845/36823477-34ed929c-1cb2-11e8-8037-7b4f4b6123b1.png)

Reply:
Replying to this comment: https://github.com/caskdata/cdap/pull/9933#discussion_r171737908
👍